punchpipe/flows/level0.py

```diff
@@ -1015,7 +1015,7 @@ def form_single_image(spacecraft, t, defs, apid_name2num, pipeline_config, space
                 cube.meta.history.add_now("form_single_image", f"Outlier detection with {limit_filename}")
                 break
             if selected_limits is None:
-                if len(outlier_limits) and file_type != 'PX':
+                if len(outlier_limits) and (file_type not in {'PX', 'DK'}):
```

Comment:
```suggestion
                if len(outlier_limits) and file_type in ['CR', 'PM', 'PZ', 'PP']:
```
Maybe we should just explicitly label the types that we expect limits for